src/Language/PureScript/Ide/State.hs

```diff
@@ -235,7 +234,7 @@ populateVolatileStateSTM ref = do
         & resolveOperators
         & resolveReexports reexportRefs
   setVolatileStateSTM ref (IdeVolatileState (AstData asts) (map reResolved results) rebuildCache)
-  pure (force results)
+  pure results
```

Comment:
I have limited haskell XP so may be misunderstanding but I suspect that by allowing results to be lazy, the IDE doesn't have to calculate all of the results as the state may be recomputed before much or any of an old result is used. 

Whereas, when it is forced, it is possible for many computations to be queued up on a large project and the `results` value can use up so much memory that the IDE starts to grind to a halt.